analyzers/src/SonarAnalyzer.CSharp/Extensions/ISymbolExtensions.cs

```diff
@@ -89,5 +91,28 @@ private static IEnumerable<SyntaxNode> GetDescendantNodes(Location location, Syn
 
             return root.DescendantNodes();
         }
+
+        /// <summary>
+        /// Returns the Identifier token of the first declaring <see cref="SyntaxReference"/>.
+        /// For e.g. an <see cref="IPropertySymbol"/> this is <see cref="PropertyDeclarationSyntax.Identifier"/>.
+        /// </summary>
+        /// <returns>The idenifier token or <see langword="null"/> if there is no DeclaringSyntaxReference or the <see cref="SyntaxNode"/> does not have an Identifier.</returns>
+        public static SyntaxToken? GetFirstIdentifier(this ISymbol symbol) =>
```

Comment:
To replace the docs:
```suggestion
        public static SyntaxToken? FirstDeclaringReferenceIdentifier(this ISymbol symbol) =>
```
I generally don't like the `Get` prefix, because everything non-void is by-definition `get` or `something and get`. And it postpones understanding the purpose 3 characters later.